ReactiveCocoa/Swift/Signal.swift

```diff
@@ -551,10 +551,27 @@ public func takeLast<T,E>(count: Int)(signal: Signal<T,E>) -> Signal<T,E> {
 	}
 }
 
+/// Forwards any values from `signal` until `predicate` returns false,
+/// at which point the returned signal will complete.
+public func takeWhile<T,E>(predicate: T -> Bool)(signal: Signal<T,E>) -> Signal<T,E> {
```

Comment:
I've seen both `<T,E>` and `<T, E>`. It seems we're using the latter but nothing is documented in the [style guide](https://github.com/github/swift-style-guide).